examples/misc/bin/cheatsheet/nullable.dart

```diff
@@ -4,13 +4,13 @@ void main() {
   {
     // #docregion invalid-null
     // ignore: invalid_assignment
-    int a = null; // INVALID in null-safe Dart.
+    int a = null; // INVALID.
     // #enddocregion invalid-null
   }
 
   {
     // #docregion valid-null
-    int? a = null; // Valid in null-safe Dart.
+    int? a = null; // Valid.
```

Comment:
To match the snippet in the cheatsheet:

```suggestion
    int? a = null; // Valid
```